github/project.go

```diff
@@ -92,13 +92,13 @@ func rawHost(host string) string {
 	}
 }
 
-func useHttpProtocol() bool {
-	https := os.Getenv("HUB_PROTOCOL")
-	if https == "" {
-		https, _ = git.Config("hub.protocol")
+func useProtocol(proto string) bool {
+	cfgproto := os.Getenv("HUB_PROTOCOL")
+	if cfgproto == "" {
+		cfgproto, _ = git.Config("hub.protocol")
```

Comment:
I'm not a fan of `cfgproto` variable. Call it something human-readable, such as `protocol` or `userProtocol`